mmcv/ops/csrc/pytorch/filtered_lrelu.cpp

```diff
@@ -0,0 +1,39 @@
+#include <cuda_runtime.h>
+#include <torch/extension.h>
+
+#include "pytorch_cpp_helper.hpp"
+#include "pytorch_device_registry.hpp"
+
+std::tuple<torch::Tensor, torch::Tensor, int> filtered_lrelu_op_impl(
```

Comment:
Parrots only support return single tensor or return by args.

Reply:
We may not support parrots for now.